src/config.ts

```diff
@@ -173,6 +174,7 @@ const DEFAULT_CONFIG: Partial<SnowpackConfig> = {
     baseUrl: '/',
     clean: false,
     metaDir: '__snowpack__',
+    optimize: true,
```

Comment:
One thought: we could do something that disables this if a user has a bundler enabled. I kinda feel like we should do this anyway in Snowpack, because if they’re manually adding a bundler they could also specify `config.buildOptions.optimize: false` too 🤷 

I’d rather have things be explicit than have hidden conditional logic where config rules do or don’t apply based on other config rules (esp. regarding a bundler)